vendor/vendor.json

```diff
@@ -182,6 +194,12 @@
 			"revisionTime": "2016-08-11T16:24:59Z"
 		},
 		{
+			"checksumSHA1": "qS3eZZfhdDM6haTZp7OeyhMDlV8=",
+			"path": "github.com/aws/aws-sdk-go/service/route53domains",
+			"revision": "f80e7d0182a463dff0c0da6bbed57f21369d4346",
```

Comment:
This revision is ~1 year old at this point (1.4.x vs 1.10.x). I needed to use this particular commit to retain compatibility with the rest of the library. I'm pretty sure problems happened with the shared parts of the SDK if I tried anything else. At what point does it make sense to update aws-sdk-go as a whole, or is it more common to update individual packages within it piecemeal?